src/beacon/drand.rs

```diff
@@ -100,16 +89,22 @@ impl BeaconSchedule {
             prev.round()
         };
 
-        let mut cur = max_round;
-        let mut out = Vec::new();
-        while cur > prev_round {
-            // Push all entries from rounds elapsed since the last chain epoch.
-            let entry = curr_beacon.entry(cur).await?;
-            cur = entry.round() - 1;
-            out.push(entry);
+        // We only ever need one entry after nv22 (FIP-0063)
```

Comment:
the quicknet change will be at exactly the same epoch as the nv22 upgrade?

Reply:
The logic and comments here replicate [lotus](https://github.com/filecoin-project/lotus/pull/11572/files#diff-587eaf0df6b60dbf741d19bbe39439f6f48ffe171e82a71c76b82484ba48f386R148), and yes, I think the planned upgrade epoch is the pineapple upgrade